### ISSUE TITLE
Setup tester fails: The main module to load is ambiguous

### ISSUE THREAD

**Issue by carymrobbins** [haskell/ghcide#467]
I'm working in a large project which contains many packages. When attempting to test ghcide from the project root I get `The main module to load is ambiguous`. If I attempt to test ghcide from one of the package roots I get the same error.

```
ghcide version: 0.0.6 (GHC: 8.6.5) (PATH: path/to/ghcide) (GIT hash: e180a626eb05aca2a0c1602022459a7c2633d6f4)
Ghcide setup tester in path/to/myproject/subproject/core.
Report bugs at https://github.com/digital-asset/ghcide/issues

Step 1/6: Finding files to test in path/to/myproject/subproject/core
Found 18 files

Step 2/6: Looking for hie.yaml files that control setup
Found 1 cradle

Step 3/6, Cradle 1/1: Implicit cradle for path/to/myproject/subproject/core
Cradle {cradleRootDir = "path/to/myproject", cradleOptsProg = CradleAction: stack}

Step 4/6, Cradle 1/1: Loading GHC Session
ghcide: CradleError (ExitFailure 1) ["Failed to parse result of calling stack","","* * * * * * * *","The main module to load is ambiguous. ....
```

**Comment by aherrmann-da** [comment]
This looks like a duplicate of https://github.com/digital-asset/ghcide/issues/328.

For a specific component you could create a `hie.yaml` file and point it to the relevant stack component, see [`hie-bios` documentation](https://github.com/mpickering/hie-bios#explicit-configuration).

Multi-cradle support is not yet implemented, you can follow the corresponding discussion here: https://github.com/digital-asset/ghcide/issues/113.

**Comment by jneira** [comment]
multi-cradle is merged and an explicit cradle should fix it (as #328), so closing
